makefile

```diff
@@ -35,6 +35,10 @@ aws:
 	@GOOS=linux GOARCH=amd64 $(BUILD_FLAGS) go build -o gollum
 	@zip -j dist/gollum_aws.zip gollum config/kinesis.conf dist/Procfile
 
+systemd:
```

Comment:
Please move linker / build options into the consumer file.
See https://github.com/trivago/gollum/blob/master/contrib/native/librdkafka/client.go#L17